config/sociallinks.js

```diff
@@ -0,0 +1,21 @@
+"use client";
+
+import { BsTwitter, BsFacebook, BsLinkedin } from "react-icons/bs";
+
+export const socialShareLinks = [
```

Comment:
- The export does not match the file name
- The file name is not camelcase 

I would suggest renaming both to be `socials`